src/main/java/se/sundsvall/myrepresentative/api/MandatesResource.java

```diff
@@ -0,0 +1,94 @@
+package se.sundsvall.myrepresentative.api;
+
+import static org.springframework.http.MediaType.*;
+
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.Parameter;
+import io.swagger.v3.oas.annotations.media.Content;
+import io.swagger.v3.oas.annotations.media.Schema;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import jakarta.validation.Valid;
+import java.util.List;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.*;
+import org.zalando.problem.Problem;
+import org.zalando.problem.violations.ConstraintViolationProblem;
+import se.sundsvall.dept44.common.validators.annotation.ValidMunicipalityId;
+import se.sundsvall.dept44.common.validators.annotation.ValidUuid;
+import se.sundsvall.myrepresentative.api.model.CreateMandate;
+import se.sundsvall.myrepresentative.api.model.MandateDetails;
+import se.sundsvall.myrepresentative.api.model.UpdateMandate;
+
+@RestController
+@RequestMapping(value = "/{municipalityId}")
+@Tag(name = "MyRepresentatives")
+@Validated
+@ApiResponse(responseCode = "400", description = "Bad request", content = @Content(mediaType = APPLICATION_PROBLEM_JSON_VALUE, schema = @Schema(oneOf = {
+	Problem.class, ConstraintViolationProblem.class
+})))
+@ApiResponse(responseCode = "404", description = "Not Found", content = @Content(mediaType = APPLICATION_PROBLEM_JSON_VALUE, schema = @Schema(implementation = Problem.class)))
+@ApiResponse(responseCode = "500", description = "Internal Server Error", content = @Content(mediaType = APPLICATION_PROBLEM_JSON_VALUE, schema = @Schema(implementation = Problem.class)))
+class MandatesResource {
+
+	@Operation(summary = "Create mandates",
+		responses = {
+			@ApiResponse(responseCode = "201",
+				description = "Created",
+				useReturnTypeSchema = true)
+		})
+	@PostMapping(value = "/mandates", consumes = APPLICATION_JSON_VALUE, produces = ALL_VALUE)
+	ResponseEntity<Void> createMandate(
+		@Parameter(name = "municipalityId", description = "Municipality id", example = "2281") @ValidMunicipalityId @PathVariable final String municipalityId,
+		@Valid @RequestBody final CreateMandate request) {
+		// TODO implement
+		return ResponseEntity.status(HttpStatus.NOT_IMPLEMENTED).build();
```

Comment:
Change to `return ResponseEntity.status(HttpStatus.NOT_IMPLEMENTED).header(CONTENT_TYPE, ALL_VALUE). build()` as this resource produces ALL_VALUE